src/index.ts

```diff
@@ -14,8 +14,8 @@
  * You should have received a copy of the GNU Lesser General Public License along with @cldn/components.
  * If not, see <https://www.gnu.org/licenses/>.
  */
-export {Component} from "./Component.js";
-export {ElementComponent} from "./ElementComponent.js";
 export {NodeComponent} from "./NodeComponent.js";
-export {SvgComponent} from "./SvgComponent.js";
 export {TextComponent} from "./TextComponent.js";
+export {ElementComponent} from "./ElementComponent.js";
+export {Component} from "./Component.js";
+export {SvgComponent} from "./SvgComponent.js";
```

Comment:
The class hierarchy must be followed when exporting. A heritage class must always be exported before the class that exports it, otherwise the code will not work.